blender/arm/make_renderpath.py

```diff
@@ -91,11 +105,19 @@ def add_world_defs():
             if rpdat.arm_voxelgi_temporal:
                 assets.add_khafile_def('arm_voxelgi_temporal')
                 wrd.world_defs += '_VoxelGITemporal'
-        wrd.world_defs += '_VoxelAOvar' # Write a shader variant
-        if rpdat.arm_voxelgi_shadows:
-            wrd.world_defs += '_VoxelShadow'
-        if rpdat.arm_voxelgi_occ == 0.0:
-            wrd.world_defs += '_VoxelAONoTrace'
+
+        if voxelgi:
+            wrd.world_defs += '_VoxelGI'
+            if rpdat.arm_voxelgi_shadows:
+                wrd.world_defs += '_VoxelGIShadow'
```

Comment:
Do we need to have a separate `_VoxelGIShadow` define? Looks like it's the same as `_VoxelShadow`.

Reply:
This is right, just that the old files did differentiate that

Reply:
What old files? Which revision should I look at? Still, if these defines are the same we should merge them into one.

Reply:
I think the voxels gi comes from 19.02